Source/Core/Core/Core.cpp

```diff
@@ -806,6 +801,17 @@ static bool PauseAndLock(bool do_lock, bool unpause_on_unlock)
   return was_unpaused;
 }
 
+static void ResetRumble()
+{
+#if defined(__LIBUSB__)
+  GCAdapter::ResetRumble();
+#endif
+#if defined(CIFACE_USE_XINPUT) || defined(CIFACE_USE_DINPUT)
+  for (int i = 0; i != 4; ++i)
```

Comment:
The typical check for this is `i < 4`.

Reply:
I'd seen in several other files that `i != 4` was used instead, which is why I used it there. Would `i < 4` be preferable?

Reply:
Using `<` for counting up to n is much more common than using `!=`, and `i != 4` is only used 3 times in the entire codebase, so yes, I think `i < 4` is preferable.